app/javascript/gobierto_visualizations/webapp/visualizations/treeMapNested.vue

```diff
@@ -229,36 +237,24 @@ export default {
         const g1 = svg
           .append("g")
           .datum(d)
-          .attr('class', d => {
-            let parentNameContract = ''
+          .attr('fill', d => {
             const { depth } = d
-            if (depth === 1) {
-              const { data: { name = '' } } = d
-              parentNameContract = name
-            } else if (depth === 2) {
-              const { parent: { data: { name = '' } } } = d
-              parentNameContract = name
-            }
-            //Normalize and create an slug because Servicios de Gestión Públicos isn't a valid class
-            parentNameContract = normalizeString(parentNameContract)
-            return `depth ${parentNameContract}`
+            const valueColor = depth === 2 ? d.color = colors(d.parent.data.name) : ''
```

Comment:
I don't understand your logic behind: You destructurate `depth`, OK good; and immediately after, you use `d.color` and `d.parent.data.name` 🤯. Try to keep consistency in the code.

Don't hurry, read your code, and keep it legible.

In fact, you'd destructured the previous commit... there are a keyboard shortcuts which they'll help you: Ctrl+X, Ctrl+V (in OSx Cmd+X, Cmd+V) 😇

Reply:
Done!